### PR TITLE
VLT-728: New instrument endpoints

### PR DESCRIPTION
**Pull request by anthony-roe-cko**
New instrument endpoints for retrieving and updating instruments along with also updating customers in UPAPI

### CODE REVIEW

File: spec/components/schemas/Instruments/RetrieveInstrumentResponse.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  - id
+  - type
+  - fingerprint
+  - expiry_month
+  - expiry_year
+  - last4
+  - bin
+properties:
+    id:
+        description: The base 32 encoded instrument id for the retrieved instrument
```
Comment (ramani-brown-cko) [inline spec/components/schemas/Instruments/RetrieveInstrumentResponse.yaml:12]: Can simplify to 'The instrument id' for consistency with other endpoints

File: spec/paths/customers@{id}.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      description: Unauthorized
+    '422':
+      description: Invalid data was sent
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ValidationError'
+    '404':
+      description: Customer not found
+
```
Comment (ramani-brown-cko) [inline spec/paths/customers@{id}.yaml:35]: Is a different response code needed for when the email address already exists for a customer of that merchant, or does it fall into one above?

Comment (anthony-roe-cko) [reply spec/paths/customers@{id}.yaml:35]: We will return a 422 with an error code in this situation and that is documented. I don't think there is anything to change here.

File: spec/components/schemas/Customers/UpdateCustomerRequest.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        type: string
+        example: JohnTest@test.com
+    name: 
+        description: The name of the customer
+        type: string
+        example: John Test
+    default: 
+        description: The instrument ID for this customer’s default instrument
+        type: string
+        example: cus_imu3wifxfvlebpqqq5usjrze6y
```
[the remark this answers is not in the document]
Comment (anthony-roe-cko) [reply spec/components/schemas/Customers/UpdateCustomerRequest.yaml]: Done

File: spec/components/schemas/Instruments/RetrieveInstrumentResponse.yaml (hunk 1 above)
Comment (anthony-roe-cko) [reply spec/components/schemas/Instruments/RetrieveInstrumentResponse.yaml]: Done

**Pull request merged**